mu4/notation/inotationelements.h

```diff
@@ -18,6 +19,7 @@ class INotationElements
     virtual ~INotationElements() = default;
 
     virtual Ms::Element* search(const std::string& searchCommand) const = 0;
```

Comment:
From that point it's not really obvious which "select" should be used. Is there any chance to merge these two methods and leave a single way to search?

Reply:
I think, better to rename this method to something like "elements" and also rename SearchElementOptions to FilterElementsOptions. By default, this method should return all score elements

Reply:
fixed. Thanks!